Python/optimizer_bytecodes.c

```diff
@@ -1088,6 +1088,17 @@ dummy_func(void) {
         res = sym_new_type(ctx, &PyLong_Type);
     }
 
+    op(_GET_LEN, (obj -- obj, len)) {
+        int tuple_length = sym_tuple_length(obj);
+        if (tuple_length == -1) {
+            len = sym_new_type(ctx, &PyLong_Type);
+        }
+        else {
+            assert(tuple_length >= 0);
+            len = sym_new_const(ctx, PyLong_FromLong(tuple_length));
```

Comment:
This leaks the new integer, and doesn't check for error:

```suggestion
            PyObject *temp = PyLong_FromLong(tuple_length);
            if (temp == NULL) {
                goto error;
            }
            len = sym_new_const(ctx, temp);
            Py_DECREF(temp);
```